README.md

```diff
@@ -15,22 +17,83 @@ Create a challenge. Ready? Brainstorm. mindwendel helps you to easily brainstorm
 ![](docs/screenshot2.png)
 
 ## Use-cases
+
 Brainstorm ...
+
 - ... new business ideas
 - ... solutions for a problem
 - ... what to eat tonight
-- ... 
+- ...
 
 ## Getting Started
 
-### Prerequisites
+mindwendel can be run just about anywhere
+
+ <!-- TODO: Add an installation guide with detailed instructions for various deployments. -->
+ <!-- So checkout our Installation Guides for detailed instructions for various deployments. -->
+
+Here's the TLDR:
+
+### Docker-Compose
+
+To run mindwendel via Docker-Compose, just type
+
+```sh
+docker-compose up
+```
+
+Note: Adjust the env vars in teh `docker-copmose.yml`.
+
+### Docker
+
+To just run mindwendel via Docker (without postgres database), just type
+
+```sh
+docker run -d --name mindwendel \
+  -p 127.0.0.1:80:4000 \
+  -e DATABASE_HOST="..." \
+  -e DATABASE_USER="..." \
+  -e DATABASE_USER_PASSWORD="..." \
+  -e DATABASE_NAME="..." \
+  -e SECRET_KEY_BASE \
+  -e URL_HOST="localhost" \
+  ghcr.io/mindwendel/mindwendel
+```
+
+NOTE: mindwendel requires a postgres database. You can use our docker-compose file to also install the postgres.
+
+## Usage example
+
+## Contributing
+
+To get started with a development installation of mindwendel, follow the instructions below.
+
+mindwendel is built on top of:
 
 - [Elixir](https://elixir-lang.org/install.html)
 - [Phoenix Framework](https://hexdocs.pm/phoenix/installation.html#phoenix)
 - [Phoenix LiveView](https://github.com/phoenixframework/phoenix_live_view)
 - [PostgreSQL](https://www.postgresql.org)
 
-### For local setup and development
+### Dev setup based on Docker
+
+- Startup docker container
+  ```sh
+  docker-compose -f docker-compose.yml -f docker-compose.dev.yml up
```

Comment:
this is a "or", right? so i would make two commands out of it.

Reply:
No. This is suppose to be just one command, see https://docs.docker.com/compose/extends/ . But it does not matter because I just removed it anyway.